src/scaraplate/rollup.py

```diff
@@ -158,6 +160,29 @@ def get_target_project_cookiecutter_context(
         return dict(context)
 
 
+def ensure_cookiecutter_context_exists(
+    target_path: Path, scaraplate_yaml: ScaraplateYaml
+) -> None:
+    cookiecutter_context = scaraplate_yaml.cookiecutter_context_type(target_path)
+    basename = Path(f"{cookiecutter_context}").relative_to(target_path)
```

Comment:
Quite abusive way to do `str(thing)`.

Reply:
This is 2019.

Reply:
Please take me back to my shiny 2008.

Reply:
The only qustion remains if that we can hive an error by such cast without worry of mypy. Technically, we shouldn't cast here anything anyway.

Reply:
`cookiecutter_context` has `__str__` which returns a full absolute path to the file. The goal of this line is to get a path relative to the temp directory.

Technically we should be fine with this, as `__str__` is considered to be a part of the public API, so it shouldn't change.

Also the result of this operation is checked in tests (which ensure that the exception message contains just `.scaraplate.conf`).

Reply:
Hm...can we avoid this magic by using somethings like `cookiecutter_context.path` which explicitly guarantee that we'll receive a path, not a some string representation?

Reply:
Pushed a different solution: https://github.com/rambler-digital-solutions/scaraplate/compare/73be435f3e95958d7a9b820d3be9f4a3feecd8a3..3171161a71fd6385ea67e53533a9a638884b4bb3

What I don't like about the previous one is that a custom implementation of CookieCutterContext might not override `__str__` or might not use paths at all, so we shouldn't assume that `__str__` is always a path.

Reply:
LGFM, thank you! :beer: